### ISSUE TITLE
after install package control, then install package , i have this issue,please have check, thanks.

### ISSUE THREAD

**Issue by BabyDiose** [wbond/package_control#999]
```
Traceback (most recent call last):
  File "./threading.py", line 901, in _bootstrap_inner
  File "package_control.package_cleanup in /Users/guowei/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 51, in run
  File "package_control.package_manager in /Users/guowei/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 1843, in record_usage
  File "package_control.console_write in /Users/guowei/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 34, in console_write
TypeError: __str__ returned non-string (type NoneType)

Exception in thread Thread-2:
Traceback (most recent call last):
  File "package_control.package_manager in /Users/guowei/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 371, in list_repositories
  File "package_control.providers.channel_provider in /Users/guowei/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 196, in get_repositories
  File "package_control.providers.channel_provider in /Users/guowei/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 87, in fetch
  File "package_control.download_manager in /Users/guowei/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 340, in fetch
  File "package_control.downloaders.urllib_downloader in /Users/guowei/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 224, in download
Package Control.package_control.downloaders.downloader_exception.DownloaderException: <unprintable DownloaderException object>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./threading.py", line 901, in _bootstrap_inner
  File "package_control.commands.install_package_command in /Users/guowei/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 46, in run
  File "package_control.package_installer in /Users/guowei/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 59, in make_package_list
  File "package_control.package_manager in /Users/guowei/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 453, in list_available_packages
  File "package_control.package_manager in /Users/guowei/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 413, in list_repositories
  File "package_control.console_write in /Users/guowei/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 34, in console_write
TypeError: __str__ returned non-string (type NoneType)
```

**Comment by guocheng** [comment]
I had the exact same issue...

I don't know if this is proxy related, but I am using shadowsocks and have this setting:

`"http_proxy": "http://127.0.0.1:1080"`

in `Package Control.sublime-settings`

**Comment by ninindi** [comment]
I had the same issue, but solved doing what explained in the following and reinstalling Package Control.
The problem was caused by the fact that the first time I installed Package Control on Sublime text 3 I also had **Sublime Text 2 open**. 
I did not notice an error that occurred at the time: 

"error: unable to open crawldb, indexing is unavailable".

So deleting the

 ~/.config/sublime-text-3/Index directory while Sublime Text is closed 

as explained here: 

https://forum.sublimetext.com/t/closed-st3-and-crawldb/11647

and reinstalling Package Control solved this issue for me.
